lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestCloseTask.cs

```diff
@@ -69,6 +74,40 @@ public void TestStopTaskOnLocalRuntime()
         }
 
         /// <summary>
+        /// This test is to close a running task and enforce it to break and return after the current iteration
+        /// </summary>
+        [Fact]
+        public void TestBreakTaskOnLocalRuntime()
```

Comment:
It is really hard as IMRU tasks require a lot injectable classes in constructors and each has nested injectable classes. It is also very complex to mock as those will be used in Call method. Too much to mock.